RELEASE-STEPS.md

```diff
@@ -69,6 +69,87 @@ The Homebrew team for Mac does not necessarily pick up Rhino releases
 automatically. It may be necessary to submit a PR to the "homebrew/homebrew"
 repo in GitHub for a change to the file "Library/Formula/rhino.rb".
 
+## Update Kangax Compatibility Table
+
+Check out `kangax/compat-table` and prepare a pull request to describe
+support in the new version of Rhino. First, add a new environment for the
+version to `environments.json` like so:
+
+```json
+"rhino1_7_14": {
+  "full": "Rhino 1.7.14",
+  "short": "Rhino 1.7.14",
+  "family": "Rhino",
+  "platformtype": "engine",
+  "release": "2022-01-06",
+  "obsolete": true
+},
+```
+
+Copy the most recent rhino JAR into the directory as `rhino.jar`. Next, run
+`rhino.js`. This will produce a number of messages indicating that failing
+tests now pass, or that new tests now have recorded results:
+
+    **** data-es2016plus ****
+
+    data-es2016plus -> Error.cause property -> AggregateError.prototype lacks cause: test result out of date, res: true, actual: false
+
+Manually update each indicated `data-*.js` file with updated test data as
+indicated. For example, a test that failed in 1.7.13 and passed in 1.7.14
+would have a `res` section with the following diff:
+
+```javascript
+   res: {
+     babel6corejs2: false,
+     babel7corejs3: babel.corejs,
+     /* ... */
+     rhino1_7_13: false,
++    rhino1_7_14: true,
+   }
+```
+
+Rerun `rhino.js` and verify it produces no output. Then `npm run build` and
+submit your pull request.
+
+## Update Babel
+
+Once the `compat-table` changes are merged, check out `babel/babel` and prepare
+a pull request to enable support for `babel-preset-env` in the new release.
+
+* Update `COMPAT_TABLE_COMMIT` in `packages/babel-compat-data/scripts/download-compat-table.sh`
+to correspond to the merge commit in `compat-table`.
+* Run `npm run build`.
+
+Then submit the resulting patch as a pull request to Babel.
+
+## Update core-js-compat
+
+Compatibility data for `core-js`, the `babel` polyfill engine, also needs to
+be updated.
+
+* Check out `zloirock/core-js` and `npm install`.
+* Copy the most recent rhino JAR into the directory as `rhino.jar`.
+* Edit `tests/compat/tests.js` by replacing all instances of `GLOBAL` with
+`global`.
+* Edit `packages/core-js-compat/src/data.js` to be a CommonJS module rather
+than a JSM. (Or find a better way to do this and edit this file.)
+* Edit `tests/compat/node-runner.js` by replacing `console.log` with `print`
```

Comment:
I think that it's better to make `rhino-runner`.

Reply:
Got it, I'll do that the next time we need to make these updates.

Reply:
This suggestion is now recorded in RELEASE_STEPS.md as a suggestion for future work, so resolving this comment